app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php

```diff
@@ -41,4 +41,35 @@ public function insertException($reportData) {
 
         $this->client->notifyOnError($reportData[0],$backtraces);
     }
+
+    /**
+     * @param string $message
+     */
+    public function insertLog($message) {
+        $messageArray = explode("\n", $message);
+        if (count($message) < 1) {
+            exit;
```

Comment:
I mean, is there a reason why we use `exit` here? Seems like `return` looks more natural, but perhaps you had a reason to use `exit`, hadn't you? :)